src/infra/azure/storage/queue/client.ts

```diff
@@ -0,0 +1,24 @@
+import { QueueServiceClient, QueueClient } from "@azure/storage-queue";
+
+import * as TE from "fp-ts/TaskEither";
+
+export const createQueueClient = (
+  connectionString: string,
+  queueName: string
+) => {
+  const queueServiceClient =
+    QueueServiceClient.fromConnectionString(connectionString);
+  return queueServiceClient.getQueueClient(queueName);
+};
+
+export const enqueueMessage = (queueClient: QueueClient) => (message: string) =>
+  TE.tryCatch(
+    () => queueClient.sendMessage(Buffer.from(message).toString("base64")),
+    () => new Error("Message cannot be queued")
```

Comment:
```suggestion
    toError
```

if you swallows errors this way we cannot have any info on why the error occurred